lib/elixir/test/elixir/module/types/expr_test.exs

```diff
@@ -210,6 +210,11 @@ defmodule Module.Types.ExprTest do
              )
   end
 
+  test "module call" do
+    assert quoted_expr(Mix.shell) == {:ok, :dynamic}
```

Comment:
I think the formatter is not going to be happy with this. You may need to use a module attribute to inject `Mix`.